Gopkg.toml

```diff
@@ -0,0 +1,3 @@
+[[constraint]]
+  name = "github.com/inconshreveable/mousetrap"
+  version = "1.0"
```

Comment:
I don't want to go ahead with `dep` anymore since go-1.11.0 go-modules are present, it would be good to start with those now.

Reply:
:+1: